testdata/p4tc_samples_outputs/direct_counter_example.template

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/direct_counter_example numtables 1
+
+$TC p4template create action/direct_counter_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/direct_counter_example/ingress/send_nh state active
+
+$TC p4template create action/direct_counter_example/ingress/drop actid 2
+$TC p4template update action/direct_counter_example/ingress/drop state active
+
+$TC p4template create extern/root/DirectCounter extid 101 numinstances 1 tc_acl 0x1136 has_exec_method 
```

Comment:
Probably the `extid` for `DirectCounter` will change according to the `p4runtime` spec
But we'll update the document with it and ping you once we do that

Reply:
The new ID is `0x1A000001` (https://github.com/p4tc-dev/p4tc-extern-interfaces/blob/main/README.md#directcounter-extern)